docs/exclusive-content.md

```diff
@@ -212,3 +204,41 @@ If you see the source files instead of the example, click **View App** in the bo
+
+The Exclusive Content Generator allows users to generate an encrypted piece of content that can be embedded on the web page.
+
+:::info
+Want to try the Exclusive Content Generator right away? Check it out [here](/exclusive-content).
+:::
+
+### How does it work?
+
+The Exclusive Content Generator is a Cloudflare worker that derives an encryption key from the user's payment pointer and a master key and returns it to the Creator's client. The client then encrypts the exclusive content and the verifier endpoint and displays all the information required for decryption in a <code>div</code> tag. It also provides a JavaScipt scipt to be embedded on the Creator's web page.
+
+[![](https://mermaid.ink/img/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG5hdXRvbnVtYmVyXG5DcmVhdG9yIC0-PiBDcmVhdG9yJ3MgQ2xpZW50OiBwYXltZW50IHBvaW50ZXIsIGNvbnRlbnQsIHZlcmlmaWVyIGVuZHBvaW50IChvcHRpb25hbClcbkNyZWF0b3IncyBDbGllbnQgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogcGF5bWVudCBwb2ludGVyXG5FeGNsdXNpdmUgQ29udGVudCBHZW5lcmF0b3IgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogZGVyaXZlcyBlbmNyeXB0aW9uIGtleSBmcm9tIHBheW1lbnQgcG9pbnRlclxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBDcmVhdG9yJ3MgQ2xpZW50OiBlbmNyeXB0aW9uIGtleVxuQ3JlYXRvcidzIENsaWVudCAtPj4gQ3JlYXRvcidzIENsaWVudDogZW5jcnlwdHMgY29udGVudCBhbmQgdmVyaWZpZXIgZW5kcG9pbnQsIGdlbmVyYXRlcyBkaXYgdGFnXG5DcmVhdG9yJ3MgQ2xpZW50IC0-PiBDcmVhdG9yOiBkaXYgYW5kIHNjcmlwdCB0YWdzIiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifSwidXBkYXRlRWRpdG9yIjpmYWxzZX0)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG5hdXRvbnVtYmVyXG5DcmVhdG9yIC0-PiBDcmVhdG9yJ3MgQ2xpZW50OiBwYXltZW50IHBvaW50ZXIsIGNvbnRlbnQsIHZlcmlmaWVyIGVuZHBvaW50IChvcHRpb25hbClcbkNyZWF0b3IncyBDbGllbnQgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogcGF5bWVudCBwb2ludGVyXG5FeGNsdXNpdmUgQ29udGVudCBHZW5lcmF0b3IgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogZGVyaXZlcyBlbmNyeXB0aW9uIGtleSBmcm9tIHBheW1lbnQgcG9pbnRlclxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBDcmVhdG9yJ3MgQ2xpZW50OiBlbmNyeXB0aW9uIGtleVxuQ3JlYXRvcidzIENsaWVudCAtPj4gQ3JlYXRvcidzIENsaWVudDogZW5jcnlwdHMgY29udGVudCBhbmQgdmVyaWZpZXIgZW5kcG9pbnQsIGdlbmVyYXRlcyBkaXYgdGFnXG5DcmVhdG9yJ3MgQ2xpZW50IC0-PiBDcmVhdG9yOiBkaXYgYW5kIHNjcmlwdCB0YWdzIiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifSwidXBkYXRlRWRpdG9yIjpmYWxzZX0)
+
+The embedded JavaScript script will parse all the exclusive content <code>div</code> tags and include the proxy payment pointer in the web page's header (if there are multiple, it will select one at random). If Web Monetization is enabled by the User, receipts can now be obtained from the <code>monetizationprogress</code> events. The receipts, together with the payment pointer and the encrypted verifier endpoint, are submitted to the Cloudflare worker, which derives the encryption key and decrypts the verifier endpoint. If the worker is able to verify the receipts with the STREAM receipt verifier, it shares the encryption key with the User's client, who is now able to decrypt the content and display it.
```

Comment:
```suggestion
The embedded JavaScript script will parse all the exclusive content <code>div</code> tags and include the proxy payment pointer in the web page's header (if there are multiple, it will select one at random). If Web Monetization is enabled by the User, receipts can now be obtained from the <code>monetizationprogress</code> events. The receipts, together with the payment pointer and the encrypted verifier endpoint, are submitted to the Exclusive Content Generator, which derives the encryption key and decrypts the verifier endpoint. If the Exclusive Content Generator is able to verify the receipts with the STREAM receipt verifier, it shares the encryption key with the User's client, who is now able to decrypt the content and display it.
```